apps/condo/domains/billing/gql.js

```diff
@@ -37,6 +37,12 @@ const BillingAccountMeterReading = generateGqlQueries('BillingAccountMeterReadin
 
 const BILLING_RECEIPT_FIELDS = `{ context ${BILLING_INTEGRATION_ORGANIZATION_CONTEXT_FIELDS} importId property { id } account { id } recipient period raw toPay printableNumber toPayDetails services ${COMMON_FIELDS} }`
 const BillingReceipt = generateGqlQueries('BillingReceipt', BILLING_RECEIPT_FIELDS)
+ 
+const BILLING_RECEIPTS_FOR_SERVICE_CONSUMER_QUERY = `
+    query getBillingReceiptsForServiceConsumer ($data: BillingReceiptsForServiceConsumerInput!) {
```

Comment:
usually, we use `where` argument inside query

Reply:
refactored!